packages/css/src/components/header/header.scss

```diff
@@ -13,29 +13,40 @@
   * Display grid is used to let both branding sections overlap.
   */
   display: grid;
+  font-family: var(--ams-header-font-family);
   padding-block: var(--ams-header-padding-block);
   padding-inline: var(--ams-header-padding-inline);
 }
 
-.ams-header__branding {
+.ams-header__logo-link {
```

Comment:
The link has turned into a grid item, which means it now takes up the full width and its clickable area covers the entire header. The other element with this class name is not a grid item, which introduces differences between the two.